application-admintools-ui/src/main/resources/AdminTools/Code/HealthCheckResources.xml

```diff
@@ -56,8 +56,7 @@
   #set ($jobId = $services.admintools.getHealthCheckJobId())
   #set ($healthJobStatus = $services.job.getJobStatus($jobId))
   {{html clean=false}}
-    &lt;div&gt;
-      &lt;span class="admin-tools-health-check-job-url" data-jobJSURL="${jobJSURL}"&gt;&lt;/span&gt;
+    &lt;div class="admin-tools-health-check-progress" data-job-js-url="${jobJSURL}"&gt;
```

Comment:
```suggestion
    &lt;div class="admin-tools-health-check-progress" data-job-js-url="${escapetool.xml($jobJSURL)}"&gt;
```

The URL shouldn't break the HTML normally, but you never know (could be a bug in ``getSkinFile``), so it's better to be on the safe side.